localstack/localstack.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM localstack/localstack
+
+# Localstack tries to connect to the host specified
+# by success_redirect_url upon successful upload of
+# content to S3. When running on a local machine,
+# this will be "localhost". Within the localstack
+# container, this doesn't point to the Civiform
+# container. In order to allow this to resolve,
+# nginx is used as a reverse proxy to forward
+# requests localhost:9000 requests to civiform:9000
+# (accessible from Docker's internal networking).
+# See https://github.com/seattle-uat/civiform/issues/2639.
+RUN apt-get install nginx --assume-yes
+
+COPY localstack.nginx.conf /etc/nginx/conf.d/
+ADD localstack-docker-entrypoint.sh /usr/local/bin/
+
+ENTRYPOINT ["localstack-docker-entrypoint.sh"]
```

Comment:
Missing newline at the end of some of these files

Reply:
Done.